python/cudf/cudf/io/parquet.py

```diff
@@ -64,11 +64,13 @@ def read_parquet(
 
 
 @ioutils.doc_to_parquet()
-def to_parquet(df, path, *args, **kwargs):
+def to_parquet(df, path, compression=None, engine="cudf", *args, **kwargs):
```

Comment:
It would be nice to add the `index` and `partition_cols` kwargs even if they're non-functional for now just for definition compatibility with Pandas.

Reply:
You got it.